src/pages/home.tsx

```diff
@@ -0,0 +1,70 @@
+export default function Home() {
+  const mock = {
```

Comment:
Lets create a mock-data folder and keep such objects there. Importing them in the components will make the code cleaner.